htdocs/bookcal/availabilities_card.php

```diff
@@ -202,8 +227,7 @@
 	}
 
 	print load_fiche_titre($langs->trans("NewObject", $langs->transnoentitiesnoconv("Availabilities")), '', 'object_'.$object->picto);
-
-	print '<form method="POST" action="'.$_SERVER["PHP_SELF"].'">';
+	print '<form method="POST" action="'.($error == 0 ? $_SERVER["PHP_SELF"] : $_SERVER["PHP_SELF"].'?action=create').'">';
```

Comment:
Do not use param action=create on url.
Into a form, you must use the param action that is a sensible parameter) as a <input> field so param does not appear into end user url submited url